packages/block-library/src/search/index.php

```diff
@@ -56,8 +56,9 @@ function render_block_core_search( $attributes ) {
 
 	if ( $show_input ) {
 		$input_markup = sprintf(
-			'<input type="search" id="%s" class="wp-block-search__input" name="s" value="%s" placeholder="%s" required />',
+			'<input type="search" id="%s" class="wp-block-search__input"%s name="s" value="%s" placeholder="%s" required />',
 			$input_id,
+			$inline_styles['shared'],
 			esc_attr( get_search_query() ),
 			esc_attr( $attributes['placeholder'] )
```

Comment:
Should we maybe put the inline styles last (or at least before `required`), and separate it with a space? Right now, the `%s` right after `class="wp-block-search__input"` could be easy to miss.

```suggestion
			'<input type="search" id="%s" class="wp-block-search__input" name="s" value="%s" placeholder="%s" %s required />',
			$input_id,
			esc_attr( get_search_query() ),
			esc_attr( $attributes['placeholder'] ),
			$inline_styles['shared']
```

Reply:
Done. 👍